associated-token-account/program/src/tools.rs

```diff
@@ -0,0 +1,67 @@
+//! Utility functions
```

Comment:
How about calling this file `create_pda_account.rs`.  Files named `tools.rs`/`utils.rs`/`common.rs` tend to collect random garbage because they have no clear purpose

Reply:
I prefer to keep general purpose utility functions in a dedicated location (incubator) so they can be later harvested and either copied to other programs or promoted to the SDK.
I agree they should have clear purpose and scope so I moved the function to `tools:account`

Reply:
Fair.  My approach is to put each function in its own file (which is why I suggested `create_pda_account.rs`), which makes it clear that nothing else belongs in that file! :)